bin/create_yml.pl

```diff
@@ -88,8 +88,22 @@
         'hemato' => {
             'institute' => 'myeloid',
             'institute_owner' => 'myeloid',
+        },
+    },
+    'mody-cftr-aatv1-0' => {
+        'rankm' => 'SNV-RM-v5.0',
+        'svrankm' => 'SV-Panel-RM-v1.0',
+        'capture_kit' => 'mody-ctfr-aatv1-0',
+        'mody' => {
+            'institute' => 'mody-test',
+            'institute_owner' => 'mody',

```

Comment:
I think Sofie has created a live institute for mody now. I guess this should be institute => 'mody-test' and institute_owner => 'MODY'

Or perhaps one would migrate the genelists from mody-test to MODY and then institute would also be 'MODY'

Reply:
OK, let's coordinate what to do here exactly before merge